src/Uno.Foundation/Metadata/ExperimentalAttribute.cs

```diff
@@ -0,0 +1,7 @@
+﻿using System;
+
+namespace Windows.Foundation.Metadata;
+
+public partial class ExperimentalAttribute : Attribute
```

Comment:
I usually copy the description from UWP docs site as `<summary>` so that the API provides description when non-UWP/WinUI is used as target framework - https://docs.microsoft.com/en-us/uwp/api/windows.foundation.metadata.experimentalattribute?view=winrt-22621 . Not crucial though 👍